source/includes/items/_create.md

```diff
@@ -0,0 +1,342 @@
+      "icon": {
+        "url": null
+      },
+      "width": null,
+      "height": null,
+      "client_id": null,
+      "is_default": false,
+      "reusable": false
+    },
+    "notes": [],
+    "task_items": [],
+    "item_attributes": [],
+    "stamps": []
+  },
+  "status": "created"
+}
+```
+
+This endpoint creates a new Item.
+
+### HTTP Request
+
+`POST http://localhost:3000/api/v6/items`
+
+### Item Parameters
+
+Parameter | Default | Description | Type | Required? | Options
+--------- | ------- | ----------- | ---- | --------- | -------
+location_id | nil | The ID of the location of the item | Integer | Yes
+condition_id | nil | The ID of the item's condition | Integer | Yes | See the Conditions endpoint
+status_id | nil | The ID of the item's status | Integer | Yes | See the Statuses endpoint
+code | nil | A free text code identifier | String | Yes
+usage | nil | Whether the Item is in use or unused | String | Yes | 'in_use', 'unused'
+project | nil | A free text project identifier | String | No
+x | nil | ??? The x position of the item on its location's floorplan. pixels or 0 to 1? ??? | Decimal | No
+y | nil | ??? The y position of the item on its location's floorplan. pixels or 0 to 1? ??? | Decimal | No
+purchase_date | nil | The purchase date | Date | No
+purchase_price_cents | nil | The purchase price of the item in pence | Decimal | No
+room | nil | A free text room identifier | String | No
+warranty | nil | The warranty length in years | Integer | No
+warranty_info | nil |  | Integer | No
+action_id | nil | an Action ID | Integer | No | See the Actions endpoint
+installation_date | nil | Installation date | Date | No
+life_expectancy | nil | The life expectancy of the item in years | Float | No
+catalogue_item_id | nil | ??? | ??? | No
+condition_details | nil | ??? Free text ??? | String | No
+
+
+### Nested Photo Parameters
+
+Parameter | Default | Description | Type | Required? | Options
+--------- | ------- | ----------- | ---- | --------- | -------
+image | nil | A photo of the item | ??? | No | ???
+
+<!-- ??? remove ???
+### Nested Task Parameters
+
+Parameter | Default | Description | Type | Required? | Options
+--------- | ------- | ----------- | ---- | --------- | -------
+completed | nil | Whether the task has been completed | Boolean | Yes |
+completed_date | nil | Date the task was completed | Date | No |
+completed_by_id | | | | | ??? needs removing from params as attribute doesn't exist on tasks ??? -->
+
+### Nested Notes Parameters
+
+Parameter | Default | Description | Type | Required? | Options
+--------- | ------- | ----------- | ---- | --------- | -------
+user_id | nil | The id of a user | Integer | Yes |
+body | nil | The note content | Text | No ??? |
+
+### Nested Item Attributes Parameters
+
+Parameter | Default | Description | Type | Required? | Options
+--------- | ------- | ----------- | ---- | --------- | -------
+catalogue_association_id | | | | |
+catalogue_attribute_id | | | | |
```

Comment:
think this needs the catalogue api to get options?

Reply:
Yup